README.md

```diff
@@ -152,6 +152,24 @@ The following includes ThreadPool related parameters, which can be passed in thr
 - whether to enable `Compare And Set`, set true if using `RawKVClient.compareAndSet` or `RawKVClient.putIfAbsent`
 - default: false
 
+### TLS
+
+#### tikv.tls_enable
+- whether to enable tls
+- default: false
+
+#### tikv.trust_cert_collection
+- Trusted certificates for verifying the remote endpoint's certificate, e.g. /home/tidb/ca.pem. The file should contain an X.509 certificate collection in PEM format.
```

Comment:
The name of this setting looks like something that could be a boolean. But this is the path to the `ca.pem` file?

We *could* consider to set this to `/etc/pki/tls/cert.pem` by default.

```
$ openssl version -d
OPENSSLDIR: "/etc/pki/tls"
```

Reply:
default is null now.

Reply:
Users can pass the argument by themself.